risc0/cargo-risczero/Cargo.toml

```diff
@@ -33,13 +33,17 @@ reqwest-middleware = "0.2"
 reqwest-retry = "0.3"
 risc0-build = { workspace = true }
 risc0-r0vm = { workspace = true, optional = true }
-risc0-zkvm = { workspace = true, optional = true }
+risc0-zkvm = { workspace = true, default-features = false, features = ["std", "getrandom"] }
+risc0-zkvm-methods = { workspace = true, default-features = false }
```

Comment:
Yes! 

I was thinking in moving the dependencies we have on `risc0-zkvm-methods` to a new crate `risc0-zkvm-methods-core` that we could publish, this way we would solve the issue. That way both `risc0-zkvm-methods` and `cargo-risczero` could depend on `risc0-zkvm-methods-core`.

Reply:
I don't think any of the benchmark binaries need any 'core' crates because the interface is very simple. We pass in the number of iterations to run, so there's no types that need to be shared between host and guest.

Reply:
Oh I see... I followed this approach because this would be similar to what we do under `/examples` having a `core` that can be used both in `guest` and `host`. 

I did the change before seeing this comment. Please let me know if you'd like to revert the new `risc0-zkvm-methods-core` and I'll change it 🙇‍♂️